selfdrive/ui/qt/onroad.cc

```diff
@@ -27,11 +27,16 @@ OnroadWindow::OnroadWindow(QWidget *parent) : QWidget(parent) {
 
   stacked_layout->addWidget(split_wrapper);
 
+  hud = new OnroadHud(this);
+  hud->setAttribute(Qt::WA_TransparentForMouseEvents, true);
+  stacked_layout->addWidget(hud);
```

Comment:
this should be part of a new stacked layout together with the nvg widget, then you can get rid of setMapWidth